asg/cloud_init.ps1

```diff
@@ -173,13 +176,22 @@ catch {
     Write-EventLog -LogName IrisAnywhere -source IrisAnywhere -EntryType Error -eventid 1001 -message "Exception establishing autologon configuration"
 }
 
+#CW Config
```

Comment:
Wondering if we should eliminate all comments in this file unless they are truly needed.  I think there is a size limit on init files we create.  Watcha think?

Another option is to have the init script just setup ENV variables then call a script that is preloaded on our AMI image.

Reply:
Im currently working on the env vars concept now. should be done in a day or so.